src/core/src/bootstrap/Constants.py

```diff
@@ -87,7 +87,7 @@ class EulaSettings(EnumBackport):
         LAST_MODIFIED = 'LastModified'
 
     TEMP_FOLDER_DIR_NAME = "tmp"
-    TEMP_FOLDER_CLEANUP_ARTIFACT_LIST = ["*.list"]
+    TEMP_FOLDER_CLEANUP_ARTIFACT_LIST = ["*.list", "azgps*"]
```

Comment:
The second addition is to remove azgps src folders added at runtime that the previous code wasn't cleaning up.